test/bucklescript/src/index_test.re

```diff
@@ -13,6 +18,9 @@ module Component = [%styled.div {|
 
   font-size: 30px;
 
+  animation-name: $(fadeIn);
+  /* animation: $(fadeIn) ease-in 200ms; */
```

Comment:
Can we add an error message if they are using the short-hand version?

Reply:
I think is easier to actually support the full version, it's probably the next development phase, developing props definition supporting variables